compatibility/prepare_backup.sh

```diff
@@ -0,0 +1,64 @@
+#!/bin/bash
+#
+# Copyright 2020 PingCAP, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set -eux
+
+# update tags
+git fetch --tags
+
+TAGS="v5.0.0"
```

Comment:
should this be changed to v4.0.something

Reply:
This Tag will be overwritten in getLatestTags, so it's ok here?🤔️

Reply:
ok 🤔